### ISSUE TITLE
The entire test rendering/compositing.py is in the skip list for parallel.

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#1037]
The entire test rendernig/compositing.py is in the skip list. I am not sure why this would fail in parallel.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 06/28/2010 07:41 pm
Original update: 08/18/2010 05:23 pm
Ticket number: 180
